push_to_3yourmind/types.py

```diff
@@ -31,15 +31,17 @@ class NoValue:
     ... it will mean that you want to set user's country, currency and language setting to NULL.
     """
 
-
+NoValueType = t.Type[NoValue]
 ResponseDict = t.Dict[str, t.Any]
 AnyResponse = t.Union[str, ResponseDict, t.List[ResponseDict], t.List[str]]
 RequestMethod = t.Literal["GET", "PUT", "POST", "DELETE", "PATCH", "HEAD"]
 Unit = t.Literal["mm", "inch"]
-CadFileSpecifier = t.Union[str, t.IO]
+AttachmentFileSpecifier = CadFileSpecifier = t.Union[str, t.IO]
 
-OptionalNumber = t.Union[int, float, decimal.Decimal, t.Type[NoValue]]
+OptionalInteger = t.Union[int, NoValueType]
+OptionalIntegerSequence = t.Sequence[OptionalInteger]
```

Comment:
are you sure? may be `OptionalIntegerSequence = t.Union[t.Sequence[int], NoValueType]`?